src/index.ts

```diff
@@ -1,8 +1,16 @@
 import {
+  ILayoutRestorer,
   JupyterFrontEnd,
   JupyterFrontEndPlugin
 } from '@jupyterlab/application';
 
+import {
+  ICommandPalette,
+  MainAreaWidget,
+  WidgetTracker
+} from '@jupyterlab/apputils';
+
+import { PyunicoreWidget, DataType } from "./pyunicoreWidget";
```

Comment:
Here we should pay attention to the server checks (e.g. user single quote) and implement those suggestions.